src/modules/emotion.command.js

```diff
@@ -0,0 +1,23 @@
+const {text, pictures} = require('./emotion-list.json');
+
+// TODO: любая эмоция может быть командой. любая команда может быть эмоцией
+// TODO: получаю эмоцию-команду и беру из пула сообщение и картинку
+const emotionCommand = async function(response, { username, input, i18n }) {
+    if (input[0] !== '!') {
+        return response;
+    }
+
+    const command = input.split(` `)[0];
+
+    const emotion = i18n(command);
+    const message = text[emotion];
+    const picture = pictures[emotion];
+
+    if (message && picture) {
+        response.output = `${message} ${picture}`;
```

Comment:
response.output only for texts... for additional data u can use something like `response.attachments = ['imageUrl']` and in client (index.js) parse and attach from this additional field

Reply:
on client-adapters side... reach data unnecessary in most cases =) in future we can send additional 'support' flags in context, but... i dont like this)

Reply:
All social via Embed, btw